Makefile

```diff
@@ -219,6 +219,10 @@ test-deps: $(PY)
 lint: $(FLAKE8)
 	$(FLAKE8) jujugui
 
+.PHONY: lint-js
+lint-js:
```

Comment:
Let's make $(NODE_MODULES) a pre-req for this. We're linting source files, not built files, so we shouldn't have to build the gui to run this.